tests/cursor/cursorinterface-001.phpt

```diff
@@ -0,0 +1,26 @@
+--TEST--
+MongoDB\Driver\CursorInterface is implemented by MongoDB\Driver\Cursor
```

Comment:
This can technically use `class_implements()` and `in_array()` without requiring a live database server (using only class name strings), but it's fine as is.

Reply:
If we'd like to test that CursorInterface extends Traversable, we can use this approach to check the interfaces directly (since we can't create concrete objects to do so).

Reply:
Done that in two new tests.